src/api/data/charts.go

```diff
@@ -8,32 +8,47 @@ import (
 	"github.com/helm/monocular/src/api/pkg/swagger/models"
 )
 
-// GetChart gets the chart associated with the passed-in repo+name
-func GetChart(repo, name string) (models.Resource, error) {
-	// TODO implement actual "get chart" business logic
+// Charts is an interface for managing chart data
+type Charts interface {
+	// will have a GetChart method to retrieve a particular chart from a repo
+	GetChart(repo, name string) (models.Resource, error)
```

Comment:
Why does `GetChart` return a `Resource` (instead of a Chart), and why does it return a `model.Resource` while the others all return `*model.Resource`?